### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Packagesreturnedforjson_Packagesreturnedforjson_Use of color : The users will not be able to distinguish between the normal text (Latest version: 13.0.1) and link (json).

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8921]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1438730

### Repro Steps

1. Hit the URL:https://www.nuget.org/
1. Tab navigate to "Sign in" control and hit enter.
1. Tab navigate to "Sign in with Microsoft" control and hit enter.
1. Search for a key term (say “json”) from search box.
1. "Packages returned for json" screen gets opened.
1. Verify and check that the users can be able to distinguish between the normal text and link.

The users will not be able to distinguish between the normal text and link in  "Packages returned for json" 

### Expected Behavior

The links should be descriptive so that users will be able to distinguish between the normal text and link.
